modules/kmm-layering-background.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/kmm-preflight-validation.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-layering-background_{context}"]
+= Layering background
+
+When a Day 0 kmod is installed in the cluster, it means that layering was applied through the Machine Config Operator (MCO) and {product-title} (OCP) upgrades won’t trigger node upgrades.
```

Comment:
```suggestion
When a Day 0 kmod is installed in the cluster, layering is applied through the Machine Config Operator (MCO) and {product-title} (OCP) upgrades do not trigger node upgrades.
```

Reply:
^generally avoid past/future tense per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) although if it needs to be past tense/future tense here for a specific reason, ignore this guidance.

Reply:
Changed.